### PR TITLE
Log Project Configuration at Process Start

### PR DESCRIPTION
**Pull request by trobanga**

### CODE REVIEW

File: clinical-domain-agent/src/main/java/care/smith/fts/cda/impl/DeidentifhirStep.java [hunk 1]
```diff
@@ -23,28 +23,30 @@
 
 @Slf4j
 class DeidentifhirStep implements Deidentificator {
+  DeidentifhirStepConfig config;
```
Comment (knoppiks) [inline clinical-domain-agent/src/main/java/care/smith/fts/cda/impl/DeidentifhirStep.java:26]: `private final`

File: clinical-domain-agent/src/test/java/care/smith/fts/cda/impl/DeidentifhirStepTest.java [hunk 2]
```diff
@@ -43,7 +45,15 @@ void setUp(MockServerClient mockServer) {
...
         new DeidentifhirStep(
-            client, domains, ofDays(14), deidentifhirConfig, scraperConfig, meterRegistry);
+            client,
+            new DeidentifhirStepConfig(
+                new TCAConfig(server, domains),
+                ofDays(14),
+                new File("IDScraper.profile"),
+                new File("CDtoTransport.profile")),
+            deidentifhirConfig,
+            scraperConfig,
```
Comment (knoppiks) [inline clinical-domain-agent/src/test/java/care/smith/fts/cda/impl/DeidentifhirStepTest.java:55]: Smells that both, the File and the parsed Config must be passed now.

File: research-domain-agent/src/test/java/care/smith/fts/rda/impl/FhirStoreBundleSenderTest.java [hunk 3]
```diff
@@ -28,6 +27,7 @@
 @ExtendWith(MockServerExtension.class)
 class FhirStoreBundleSenderTest {
 
+  FhirStoreBundleSenderConfig config;
```
Comment (knoppiks) [inline research-domain-agent/src/test/java/care/smith/fts/rda/impl/FhirStoreBundleSenderTest.java:30]: `private`

**Review by knoppiks — CHANGES_REQUESTED** [review]

File: clinical-domain-agent/src/test/java/care/smith/fts/cda/impl/DeidentifhirStepTest.java (hunk 2 above)
Comment (trobanga) [reply clinical-domain-agent/src/test/java/care/smith/fts/cda/impl/DeidentifhirStepTest.java:55]: Yes, I was thinking about moving the config parsing into DeidentifhirStep

**Review by knoppiks — CHANGES_REQUESTED** [review]

**Review by knoppiks — APPROVED** [review]

**Pull request merged**